ff2mpv.bat

```diff
@@ -1,2 +1,2 @@
 @echo off
-call python ff2mpv.py
```

Comment:
Please re-add the trailing newline here.

Reply:
There is none in the original

Reply:
Which original? `master` has a trailing newline on it, and that's intentional.